src/reformat_libyuv.c

```diff
@@ -255,49 +221,37 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
     // Find the correct libyuv YuvConstants, based on range and CP/MC
     const struct YuvConstants * matrixYUV = NULL;
     const struct YuvConstants * matrixYVU = NULL;
-    if (image->yuvRange == AVIF_RANGE_FULL) {
+    if (image->yuvRange == AVIF_RANGE_LIMITED) {
```

Comment:
i think the reason behind the changes in this function is to order the if checks in the enum order?

if so, it could be a separate PR.

Reply:
In general, handling enum values in the enum order makes it easier to verify all enum values are handled. In this particular case, the order doesn't matter because it's clear that range is either limited or full. This is why I think it's not necessary to reorder this if-else statement.

Reply:
yeah i am okay with not re-ordering this as well.